image/db/rhel/konflux.Dockerfile

```
@@ -50,3 +44,22 @@ EXPOSE 5432
 CMD ["postgres", "-c", "config_file=/etc/postgresql.conf"]
 
 USER 70:70
+
+FROM scanner-db-common AS scanner-db-slim
+
+LABEL \
+    com.redhat.component="rhacs-scanner-db-slim-container" \
+    io.k8s.display-name="scanner-db-slim" \
+    name="rhacs-scanner-db-slim-rhel8"
+
+ENV ROX_SLIM_MODE="true"
+
+FROM scanner-db-common AS scanner-db
+
+LABEL \
+    com.redhat.component="rhacs-scanner-db-container" \
+    io.k8s.display-name="scanner-db" \
+    name="rhacs-scanner-db-rhel8"
+
+COPY blob-pg-definitions.sql.gz \
+     /docker-entrypoint-initdb.d/definitions.sql.gz
```

Continuing the thread from the previous PR <https://github.com/stackrox/scanner/pull/1436#discussion_r1520988886>. I don't think it is _necessary_ to maintain root ownership because I don't think it adds a lot of protection in k8s where we are _supposed_ to run containers with `readOnlyRootFilesystem: true`. The situation will be different if we run the container in `docker`/`podman`, e.g. in some integration tests (which we don't do).

I think, file permissions and ownership in Scanner are a bit of a mess and here we diverge a bit from downstream. Just for the sake of keeping things the same until someone cleans up the mess I suggest to explicitly set former ownership. Looking at your other examples, it seems could be more simpler achieved with

```suggestion
COPY --chown=0:0 blob-pg-definitions.sql.gz \
     /docker-entrypoint-initdb.d/definitions.sql.gz
```